package.json

```diff
@@ -126,23 +128,25 @@
     "instantsearch.js": "3.1.0",
     "jest": "23.6.0",
     "jest-preset-angular": "6.0.2",
-    "ng-packagr": "3.0.6",
+    "ng-packagr": "4.7.0",
     "node-sass": "4.11.0",
     "prettier": "1.13.7",
     "raw-loader": "1.0.0",
     "readline-sync": "1.4.9",
     "replace-in-file": "3.4.3",
-    "rxjs": "5.5.12",
+    "rxjs": "6.4.0",
     "semver": "5.6.0",
     "shelljs": "0.8.3",
     "style-loader": "0.23.1",
     "ts-loader": "5.3.3",
     "tsickle": "0.34.0",
+    "tslib": "1.9.0",
     "tslint": "5.12.1",
     "tslint-config-airbnb": "5.11.1",
     "tslint-config-prettier": "1.17.0",
     "tslint-plugin-prettier": "2.0.1",
     "typescript": "3.2.4",
+    "webpack": "4.3.0",
```

Comment:
what's the reason we are adding webpack here. Could we instead leverage ["resolutions" (the Yarn feature)](https://yarnpkg.com/en/docs/selective-version-resolutions#search)

Reply:
it's a peerDependency of css-loader, raw-loader, ... which I don't think we need actually.
let's me check if I get rid of them.

Reply:
removed it along with all the dependencies that turned out to be unsed https://github.com/algolia/angular-instantsearch/pull/470/commits/c5b6b3ce2ffb2e3bf1f22d46b767ce5723e6193d